jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/compression/HuffmanEncoder.java

```diff
@@ -90,7 +89,7 @@ private static int octetsNeeded(final int[][] table, String s)
         for (int i = 0; i < len; i++)
         {
             char c = s.charAt(i);
-            if (HttpTokens.isIllegalFieldVchar(c))
+            if (c >= EOS)
```

Comment:
having a HttpTokens.isXyx method is clearer than whatever this test is doing

Reply:
But this has nothing to do with HttpTokens, the Huffman table defined by rfc7541 allows you to encode/decode 0-255 so EOS is 256 which is not able to be encoded.

Reply:
Then a `Huffman.isXyz(c)` method would read better.   Is >= EOS good or bad?